app/views/messages/_message_summary.html.erb

```diff
@@ -1,13 +1,13 @@
-<tr id="inbox-<%= message_summary.id %>" class="inbox-row<%= "-unread" unless message_summary.message_read? %>">
+<tr id="inbox-<%= message_summary.id %>" class="message-summary inbox-row<%= "-unread" unless message_summary.message_read? %>">
   <td><%= link_to message_summary.sender.display_name, user_path(message_summary.sender) %></td>
   <td><%= link_to message_summary.title, message_path(message_summary) %></td>
   <td class="text-nowrap"><%= l message_summary.sent_on, :format => :friendly %></td>
   <td class="text-nowrap d-flex justify-content-end gap-1">
-    <%= button_to t(".unread_button"), message_mark_path(message_summary, :mark => "unread"), :remote => true, :class => "btn btn-sm btn-primary", :form => { :class => "inbox-mark-unread", :hidden => !message_summary.message_read? } %>
-    <%= button_to t(".read_button"), message_mark_path(message_summary, :mark => "read"), :remote => true, :class => "btn btn-sm btn-primary", :form => { :class => "inbox-mark-read", :hidden => message_summary.message_read? } %>
-    <%= button_to t(".destroy_button"), message_path(message_summary, :referer => request.fullpath), :method => :delete, :remote => true, :class => "btn btn-sm btn-danger", :form_class => "inbox-destroy" %>
+    <%= button_to t(".unread_button"), message_mark_path(message_summary, :mark => "unread"), :class => "btn btn-sm btn-primary", :form => { :"data-turbo" => true, :class => "inbox-mark-unread", :hidden => !message_summary.message_read? } %>
```

Comment:
I'm not a fan of `:"..."` symbols I have to say... Can this be done as `:data => { :turbo => true }` which I think is how we've normally done data, or does that not work in this context?

Reply:
🤔 I think in this context it should work as long as the `:data => { :turbo => true }`-structure is passed into the `:form => …`-key.

This would be:

```
:form => { :data => { :turbo => true }, :class => "inbox-mark-unread", …
```

I deliberately wanted to avoid more nesting in the hash structure, but I'd prefer to keep it consistent with the pre-existing style in the project. I'll have a look for other places in the partials and will report back.

Reply:
see 2a2a317

Reply:
In other templates similar attributes are nested like `:data => { :foo => "bar" }`, e.g. here:

- https://github.com/openstreetmap/openstreetmap-website/blob/8da80ff471f7d04984ff5abab0c5fe32920330ba/app/views/layouts/_search.html.erb#L24
- https://github.com/openstreetmap/openstreetmap-website/blob/8da80ff471f7d04984ff5abab0c5fe32920330ba/app/views/traces/show.html.erb#L66
… and more places.

So for consistency reasons I agree to nest the data-attributes here, too. See 2a2a317